caffe2/core/operator_c10wrapper.h

```diff
@@ -56,7 +56,7 @@ class C10OperatorWrapper final : public Operator<Context> {
   }
 
   size_t num_inputs() {
-    return op_.schema().arguments().size() - num_output_parameters - num_parameters();
+    return op_.schema().arguments().size() - 1 - num_parameters();
```

Comment:
The magic constant 1 here seems horribly underjustified

Reply:
true. Will change this in the diff stacked on top.